modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl

```diff
@@ -81,10 +81,15 @@ echo "${forseti_environment}" > /etc/profile.d/forseti_environment.sh | sudo sh
 gsutil cp gs://${storage_bucket_name}/configs/forseti_conf_server.yaml ${forseti_server_conf_path}
 gsutil cp -r gs://${storage_bucket_name}/rules ${forseti_home}/
 
+# Download the Newest Config Validator constraints from GCS
+rm -rf /home/ubuntu/policy-library
+gsutil cp -r gs://{scanner_bucket}/policy-library /home/ubuntu/
```

Comment:
Where are the `policy-library` files populated from?

Reply:
This is a step user would need to do manually, to upload the policy-library folder. It will be part of the documentation for the config-validator scanner